tests/sles4sap/saptune/mr_test.pm

```diff
@@ -50,9 +54,15 @@ sub setup {
     #   Run 'zypper ps -s' to list these programs."
     zypper_call "in python3-rpm", exitcode => [0, 106];
     # Download mr_test and extract it to $HOME
-    assert_script_run "curl -sk $tarball | tar zxf - --strip-components 1";
+    assert_script_run "curl -sk $tarball | tar zxf - --strip-components 1" unless get_var('PUBLIC_CLOUD_SLES4SAP');
     # Add $HOME to $PATH
     assert_script_run "echo 'export PATH=\$PATH:\$HOME' >> /root/.bashrc";
+    # Add '/root' to $PATH for public cloud instance
+    if (get_var('PUBLIC_CLOUD_SLES4SAP')) {
+        assert_script_run "echo 'export PATH=\$PATH:/root' >> /root/.bashrc";
```

Comment:
can you elaborate why you need to add root to PATH ?

Reply:
Under non public cloud env the $HOME is root; So in order to keep the same testing environment with non public cloud env, I added "/root" to $PATH.